src/Badge.webmodeler.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, createElement } from "react";
+import { Badge, BadgeProps } from "./components/Badge";
+import { Overlay } from "./components/Overlay";
+import BadgeContainer, { BadgeContainerProps } from "./components/BadgeContainer";
+
+declare function require(name: string): string;
+
+export class preview extends Component<BadgeContainerProps, {}> {
+
+    componentWillMount() {
+        const css = require("./ui/Badge.css");
+        this.addPreviewStyle(css, "widget-badge");
+    }
+
+    render() {
+        return createElement(Overlay, {},
+            createElement(Badge, this.transformProps(this.props))
+        );
+    }
+    private transformProps(props: BadgeContainerProps): BadgeProps {

```

Comment:
space missing between function declarations. Put space between `render` and `transformProps`